### PR TITLE
Add mousify version to neurodamus toolchain

### PR DESCRIPTION
**Pull request by pramodk**

### CODE REVIEW

**Comment by pramodk** [comment]
note: do not merge as we are using temporary branch here.

File: var/spack/repos/builtin/packages/synapsetool/package.py [hunk 1]
```diff
@@ -50,6 +50,7 @@ class Synapsetool(CMakePackage):
     depends_on('highfive~mpi', when='~mpi')
     depends_on('mpi', when='+mpi')
     depends_on('python', when='+python')
+    depends_on('libsonata', when='+sonata')
```
Comment (ferdonline) [inline var/spack/repos/builtin/packages/synapsetool/package.py:53]: This should definitely be merged to develop

**Review by ferdonline — APPROVED** [review]
Changes to synapsetool should be merged. The other changes, as needed, as I have nothing against them

File: var/spack/repos/builtin/packages/neurodamus-base/package.py [hunk 2]
```diff
@@ -34,6 +34,7 @@ class NeurodamusBase(Package):
     url      = "ssh://bbpcode.epfl.ch/sim/neurodamus/bbp"
 
     version('master',      git=url, branch='master')
+    version('mousify',     git=url, branch='sandbox/leite/mousify')
```
Comment (pramodk) [inline var/spack/repos/builtin/packages/neurodamus-base/package.py:37]: only this version is concern but I will merge this for now as we need this on BB5

**Pull request merged**